sw/device/lib/arch/device_sim_verilator.c

```diff
@@ -18,6 +18,10 @@ const device_type_t kDeviceType = kDeviceSimVerilator;
 // `test/systemtest/earlgrey/test_sim_verilator.py`.
 const uint64_t kClockFreqCpuHz = 500 * 1000;  // 500kHz
 
+// The division by 2 is tied to the frequency above. Notice since the cycle
```

Comment:
I wonder whether it would be a good idea to have a static assertion check that `kClockFreqCpuHz < 1Mhz` adding a friendly message informing the need to refactor the `to_cpu_cycles` function.

Reply:
```
#include <assert.h>

static_assert(kClockFreqCpuHz < 1000000, "The verilator simulation assumes the CPU frequency is less than 1MHz");
```

Reply:
I assume <assert.h> and static_assert will do.

Reply:
Done